### PR TITLE
Change favicon in live log dynamically

### PR DESCRIPTION
**Pull request by hellcp**
Changes favicon to match the current build status
Screenshot from 2019-07-16 20-12-41
Screenshot from 2019-07-16 20-12-46
Screenshot from 2019-07-16 20-12-54

### CODE REVIEW

File: src/api/app/views/webui2/webui/package/live_build_log.html.haml [hunk 1]
```diff
@@ -35,5 +35,8 @@
       = render partial: 'live_build_log_controls'
 
 :javascript
+  faviconFinished = '#{path_to_image('favicon-warning.ico')}'
+  faviconRunning = '#{path_to_image('favicon-info.ico')}'
+  faviconPaused = '#{path_to_image('favicon.ico')}'
```
Comment (danidoni) [inline src/api/app/views/webui2/webui/package/live_build_log.html.haml]: How about passing all those paths as new parameters in the LiveLog constructor below? That way, there's no need to define global variables and the jslinter will not complain.

Please, remember to make the same change in the previous `live_build_log.html.erb`.

Comment (hellcp) [reply src/api/app/views/webui2/webui/package/live_build_log.html.haml]: Good point, sounds saner than the current way :smile:

Comment (hennevogel) [reply src/api/app/views/webui2/webui/package/live_build_log.html.haml]: @hellcp do you think you find the time to go over it?

**Comment by hennevogel** [comment]
coveralls hangs, see codecov. merging.

**Comment by hennevogel** [comment]
@hellcp awesome!

**Pull request merged**